website/templates/base.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
+
+        <title>ZoomFront</title>
+    </head>
+    <h1> ZoomFront </h1>
+    <ul>
```

Comment:
Around here is where you'd want to add the call to `get_flashed_messages()` if you wanted to add them in, since all your other templates extend this one!